README.md

```diff
@@ -0,0 +1,139 @@
+  retired, the number of CPU cyles, and fine-grained CPU frequency data. This
+  helps expose how efficient workload execution is on the hardware.
+
+* **High frequency sampling**: For traditional telemetry sources, Rezolus will
+  sample at high frequency and provide summary statistics about behavior over
+  time. By capturing data at high resolution, Rezolus is able to see spikes in
+  utilization. Normally more data makes it more expensive to collect and store
+  in your observability stack. But with Rezolus' summary statistics, these
+  spikes will be reflected in percentile summaries. It's cheaper to export a few
+  percentiles for each metrics than it is to collect on a secondly (or finer)
+  basis.
+
+## Overview
+
+Rezolus is designed to produce high resolution systems performance telemetry. It
+has a collection of samplers which instrument various aspects of systems
+performance including CPU, GPU, task scheduling, system calls, TCP, block IO,
+and more.
+
+All of Rezolus's sampler focus on capturing key signals that can be used to
+understand how the workload is running on the underlying system. These insights
+are useful for understanding where bottlenecks and optimization opportunities
+might be. With high frequency sampling and eBPF, Rezolus can also provide
+insights into your workload itself like what typical block and network IO sizes
+are, the number and type of system calls being executed, and if there's spikes
+in utilization metrics.
+
+Rezolus provides valuable data about systems performance and can be used to root
+cause production performance issues, capture better data in test environments,
+and provide signals for optimization efforts.
+
+## Getting Help
+
+If you have a problem using Rezolus or a question about Rezolus that you can't
+find an answer to, please open a
+[new issue on GitHub][new issue]
+
+## Building
+
+Rezolus is built using the Rust toolchain. If you do not have the Rust toolchain
+installed, please see [rust-lang.org][https://www.rust-lang.org/] to get
+started with Rust.
+
+### Build Dependencies
+
+For a default build of Rezolus, in addition to the rust toolchain, you will
+need:
+
+* clang >= 11.0
+
+To build with eBPF support, the following additional dependencies are required:
+
+* libelf-dev >= 0.183
+* make >= 4.3
+* pkg-config >= 0.29.2
+
+Debian and Ubuntu users can install all the required dependencies with:
+
+```bash
+sudo apt install clang libelf-dev make pkg-config
+```
+
+### Steps
+
+* clone this repository or transfer the contents of the repository to your build
+  machine
+* change directory into the repository root
+* run `cargo build` in release mode, optionally enabling the bpf feature
+
+```bash
+git clone https://github.com/iopsystems/rezolus
+cd rezolus
+cargo build --release --features bpf
+```
+
+### Configuration
+
+At this time, the configuration for Rezolus is very minimal with the only option
+being the listen address of the agent. See the [config.toml file][config] for
```

Comment:
`[config.toml file][config]` Do you intend to provide a (relative) hyperlink here?

Reply:
Fixed.